### PR TITLE
gh issue template: Add instructions for author changes

### PR DESCRIPTION
**Pull request by akoehn**
This is getting out of hand, let's give people an even more explicit pointer regarding author changes.

### CODE REVIEW

**Comment by akoehn** [comment]
Anyone willing to ACK this?

File: .github/ISSUE_TEMPLATE/file-a-correction.md [hunk 1]
```diff
@@ -11,7 +11,16 @@ assignees: mjpost
 
 [See here](https://www.aclweb.org/anthology/info/corrections/) to read about the three types of corrections.
 
-## Metadata correction: please describe the issue here**
+## Changes to the author list
+
+you need permission from the relevant conference program chairs if you
+want to change the author list in the metadata, see
```
Comment (mjpost) [inline .github/ISSUE_TEMPLATE/file-a-correction.md:17]: Capitalize you?

**Review by mjpost — APPROVED** [review]
I was going to wait to fold this in with #1273, but we can just merge this now.

**Comment by akoehn** [comment]
thx, will squash-merge

**Pull request merged**